### PR TITLE
Add updates to README and warmup period to run_vegeta.py

### PR DESCRIPTION
**Pull request by LucasTecton**
Updated the README with some requested updates

Also, added a 5-second warmup period to the `run_vegeta.py` script, so that p99 latency isn't inflated.

### CODE REVIEW

**Comment by LucasTecton** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1** <a href="https://app.graphite.dev/github/pr/tecton-ai/benchmark-selfserve/1" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/tecton-ai/benchmark-selfserve/1?utm_source=stack-comment).

File: README.md [hunk 1]
```diff
@@ -73,30 +77,56 @@ to apply the changes to your cluster:
...
+`./run_vegeta.py` commands into your console.
 
 Big red button to kill all vegeta load tests:
 ```
 ps aux | grep vegeta | awk '{print $2}' | xargs kill
 ```
 
+### Caveat
+Load tests in production clusters **will affect your production traffic**, even if your
+your feature services are in a separate workspace. If you'd like to scale your capacity
```
Comment (jackwellsxyz) [inline README.md]: I might make this stronger. Something like "If you are unsure if your cluster can handle the load capacity, please reach out to your Tecton points of contact to discuss and provision appropriately."

**Review by jackwellsxyz — COMMENTED** [review]
lgtm added a comment but otherwise, fine by me.

File: README.md (hunk 1 above)
Comment (LucasTecton) [reply README.md]: Added!

**Review by jackwellsxyz — APPROVED** [review]

**Pull request merged**